### DISCUSSION TITLE
Casambi Classic

### DISCUSSION THREAD

**Discussion by krangchen** [lkempf/casambi-bt-hass#58] [Q&A]
Dear lkempf,

Great project!
I tried to do the same trick by throwing money at the problem purchasing a WAY overprized Casambi Gateway (400 Bucks for an somethng like a pi zero poe ...) but that didn't work 'cause I'm stuck to >50 well functioning but classic Casambi lights/switches...
Your project works well for a test on the one Evo switch I have so I was wondering if you plan to help those old house owners as well?

Again thanks for your contribution!

**Comment by lkempf** [comment]
There's an open issue for that under https://github.com/lkempf/casambi-bt-hass/issues/6
I've looked into it a bit but it was quite complex and I haven't found the time to really investigate it
